ntbs-service/Pages/LegacyNotifications/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page "/LegacyNotifications/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.LegacyNotifications.Index
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
+                <div class="@notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+    </nhs-grid-row>
+    
+    @if (Model.LegacyImportResult == null)
+    {
+        <div class="ondemand-migration-description-container">
```

Comment:
Possibly want to get rid of the br tags now that we have p tags and put one br tag between the banner and the copy text